sbin/common-functions.sh

```diff
@@ -49,6 +50,22 @@ checkingAndDownloadingAlsa()
   fi
 }
 
+# Freemarker for OpenJ9
```

Comment:
We have a PR related to this, will the below be also integrated there?

Reply:
You're referring to #144 for freetype, I guess? I hadn't noticed that one until now. Would you be ok with doing that work as a follow-on enhancement? I'd really like to get this one folded into the master branch, but if you feel strongly about it, I will look into applying the #144 approach for freemarker here.

Reply:
Yes exactly, feel free to use the script(s) there and we can work together to iron it out. That way there would be consistency across the different scripts. Happy to help you out, if you get stuck or need any explanation on what we are doing. 

And doing it as a follow-on work is fine, as long it does not fall thru. :+1: